runtime/doc/options.txt

```diff
@@ -6816,6 +6816,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 
 	Level   Messages ~
 	----------------------------------------------------------------------
+	1	Enables Lua tracing (see above). Does not produce messages.
 	2	When a file is ":source"'ed, or |shada| file is read or written.
```

Comment:
@zeertzjq seems intuitive/helpful to have an item for level 1

Reply:
Context #27126 #27129